src/services/headerbidding.js

```diff
@@ -118,17 +122,6 @@ export function fetchBids({
         });
       });
   } else {
-    setTimeout(() => {
-      fetchBids({
-        ad,
-        id,
-        slotName,
-        dimensions,
-        wrapper,
-        bidding,
-        correlator,
-        prerender
-      });
-    }, 200);
+    setTimeout(() => initializeBiddingServices(), 200);
```

Comment:
instead of fetching bids right away we now wait until the bid services have been initalized